src/app/BUILD.gn

```diff
@@ -225,14 +225,17 @@ static_library("interaction-model") {
 
   deps = [ "${chip_root}/src/app:events" ]
 
+  # Temporary dependency: codegen data provider instance should be provided
+  # by the application
```

Comment:
Where is the issue tracking this?  It should be linked here.

Reply:
#36230 is the generic coverage for these updates.